utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/confluent/formatter/UniversalToJsonFormatterFactory.scala

```diff
@@ -78,8 +79,8 @@ class UniversalToJsonFormatter[K: ClassTag, V: ClassTag](kafkaConfig: KafkaConfi
     val deserializedRecord = deserializeRecord(record, messageWithSchemaId = valueSchemaIdOpt.isDefined)
 
     val serializableRecord = UniversalSerializableConsumerRecord(
-      keySchemaIdOpt,
-      valueSchemaIdOpt,
+      keySchemaIdOpt.map(_.asInt),
+      valueSchemaIdOpt.map(_.asInt),
```

Comment:
maybe `UniversalSerializableConsumerRecord` should depend on `SchemaId` not int? so it can be reused for integration with Azure?

Reply:
done